packages/manager/apps/pci-kubernetes/src/components/VersionSelector.component.tsx

```diff
@@ -0,0 +1,89 @@
+import { useEffect, useState } from 'react';
+import clsx from 'clsx';
+import {
+  OsdsChip,
+  OsdsSpinner,
+  OsdsText,
+  OsdsTile,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_CHIP_SIZE,
+  ODS_SPINNER_SIZE,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import { useGetCloudSchema } from '@/api/hooks/useCloud';
+
+export const tileClass =
+  'cursor-pointer border-[--ods-color-blue-100] hover:bg-[--ods-color-blue-100] hover:border-[--ods-color-blue-600]';
+
+export const selectedTileClass =
+  'font-bold bg-[--ods-color-blue-100] border-[--ods-color-blue-600]';
+
+export interface VersionSelectorProps {
+  onSelectVersion: (version: string) => void;
+}
+
+export function VersionSelector({
+  onSelectVersion,
+}: Readonly<VersionSelectorProps>) {
+  const { t } = useTranslation('versions');
+  const { data: schema, isPending } = useGetCloudSchema();
+  const [selectedVersion, setSelectedVersion] = useState('');
+  const versions = schema?.models['cloud.kube.VersionEnum'].enum || [];
+  const lastVersion = [...versions].pop();
+
+  const selectVersion = (version: string) => {
+    if (version) {
+      setSelectedVersion(version);
+      onSelectVersion(version);
+    }
+  };
+
+  useEffect(() => {
+    selectVersion(lastVersion);
+  }, [lastVersion, setSelectedVersion]);
```

Comment:
Why using the setter function instead of the value here ?
```suggestion
  }, [lastVersion, selectedVersion]);
```

Reply:
selectedVersion was useless as dependency, i removed it